service/pom.xml

```diff
@@ -291,8 +291,9 @@ under the License.
                     </execution>
                 </executions>
                 <configuration>
+                    <classifier>exec</classifier>
                     <fork>true</fork>
-                    <layout>JAR</layout>
+                    <layout>ZIP</layout>
```

Comment:
we are modifying existing configuration here -- does it mean that any **existing** artifacts will change their semantics?

Reply:
will this change be required, if we'll do something like that:
```
java -cp griffin-service.jar -Dloader.path="/path/to/lib/plugin.jar" org.springframework.boot.loader.PropertiesLauncher
```
?